src/base/preferences.cpp

```diff
@@ -1552,6 +1562,8 @@ void Preferences::setSpeedWidgetGraphEnable(int id, const bool enable)
 
 void Preferences::upgrade()
 {
+    SettingsStorage *ss = SettingsStorage::instance();
```

Comment:
@Couchy:  Is there a particular reason why you add this change?  

I know that I, personally, would rather see the original `SettingsStorage::instance()` instead of an unreadable shortcut like `ss`.

Reply:
Calling SettingsStorage::instance() multiple times just seemed redundant to me.

Reply:
Then name it "settingsStorage" (or at least "settings").